packages/react/package.json

```diff
@@ -68,6 +69,7 @@
     "eslint-plugin-react-hooks": "^4.6.0",
     "eslint-plugin-react-refresh": "^0.4.4",
     "jsdom": "^24.0.0",
+    "lightningcss": "^1.23.0",
```

Comment:
If we're not using lightningcss, let's remove the dependency and re-install to update the lock file